tests/api/utils/singletons/test_k8s_utils.py

```diff
@@ -68,11 +68,12 @@ def test_get_logger_pods_label_selector(
     if extra_selector:
         selector += f",{extra_selector}"
 
-    k8s_helper = server.api.utils.singletons.k8s.K8sHelper(namespace, silent=True)
     k8s_helper.list_pods = unittest.mock.MagicMock()
```

Comment:
is this needed after you added the fixture?

Reply:
@alonmr What do you mean? if we need to mock list_pods? yes